citesphere/src/main/webapp/WEB-INF/views/home.html

```diff
@@ -1,5 +1,59 @@
 <html layout:decorate="~{layouts/main}">
 <head>
+<script th:src="@{/resources/notify/bootstrap-notify.min.js}"></script>
+<link th:href="@{/resources/notify/animate.css}" rel="stylesheet"/>
+
+<script th:inline="javascript">
+
+$(function() {
+	$(".delete-link").click(function(event) {
+		var id = $(this).data('group-id');
+		$("#delete-client-button").attr("data-group-id", id);
+		$("#confirm-client-entry-id").text(id);
+		$('#client-delete-confirmation').modal("show");	
+		event.preventDefault();
+	});
+	$("#delete-client-button").click(function() {
+		var id = $(this).data("group-id");
+		var baseurl = 'auth/group/' + id + '/resync';
+		$.ajax({
+			'url': baseurl + "?[(${_csrf.parameterName})]=[(${_csrf.token})]",
+			'type': "POST",
+			'success': function(data) {
+				$.notify('<i class="fas fa-check-circle"></i> Successfully deleted!', {
```

Comment:
Should say "Local copy of group has been deleted. Re-sync will start as soon as group is accessed again."